spec/mailers/candidate_mailer_spec.rb

```diff
@@ -384,14 +384,32 @@
     end
 
     describe '.interview_updated' do
-      let(:email) { mailer.interview_updated(application_choice_with_interview, interview) }
+      let(:course_option) do
+        create(
```

Comment:
do we need to define this? I think these values are all stubbed and defined and the beginning of the spec?